hugo/content/overview.md

```diff
@@ -17,13 +17,13 @@ NGINX Agent is a companion daemon for your NGINX Open Source or NGINX Plus insta
 - Notifications of NGINX events
 
 
-{{< img src="docs/grafana-dashboard-example.png" caption="Grafana  dashboard showing metrics reported by NGINX Agent" alt="Grafana dashboard showing metrics reported by NGINX Agent" width="99%">}}
+{{< img src="grafana-dashboard-example.png" caption="Grafana  dashboard showing metrics reported by NGINX Agent" alt="Grafana dashboard showing metrics reported by NGINX Agent" width="99%">}}
 
```

Comment:
The grafana-dashboard-example.png should be moved as part of this?

Reply:
There are 2 instances of this and the other file mentioned. Only one set belongs to the hugo docs and is moving